src/replication_test.c

```diff
@@ -614,6 +622,9 @@ main(int argc, char **argv)
 						} else if(((uint64_t)count < total_len - recv_len) && errno == EAGAIN) {
 							recv_len += count;
 							break;
+						} else if (count == 0) {
```

Comment:
this should be after initial 'if' check for count < 0..
same thing with next chunk of changes also

Reply:
ok